packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.html

```diff
@@ -234,34 +234,85 @@
-                                items="$ctrl.autoCompletePostCode"
-                                model="$ctrl.directoryForm.postCode"
-                                name="contactPostCode"
-                                id="contactPostCode"
-                                data-on-change="$ctrl.onPostCodeChange()"
-                                required
-                                searchable
-                            >
-                            </oui-select>
-                        </oui-field>
+                        <div class="col-md-6">
+                            <div class="row">
+                                <oui-field
+                                    data-ng-if="!$ctrl.isServiceInUK"
+                                    label="{{:: 'telephony_service_contact_zipcode' | translate }}*"
+                                    error-messages="{ 'required': ('telephony_service_contact_must_fill' | translate) }"
+                                    class="col-md-6"
+                                >
+                                    <input
+                                        data-ng-if="$ctrl.autoCompletePostCode.length === 0"
+                                        type="text"
+                                        id="contactPostCode"
+                                        name="contactPostCode"
+                                        class="oui-input"
+                                        required
+                                        data-ng-change="$ctrl.onPostCodeChange()"
+                                        data-ng-model="$ctrl.directoryForm.postCode"
+                                        data-ng-model-options="{ debounce: 500 }"
+                                    />
+                                    <oui-select
+                                        data-ng-if="$ctrl.autoCompletePostCode.length > 0"
+                                        items="$ctrl.isServiceInUKautoCompletePostCode"
+                                        model="$ctrl.directoryForm.postCode"
+                                        name="contactPostCode"
+                                        id="contactPostCode"
+                                        data-on-change="$ctrl.onPostCodeChange()"
+                                        required
+                                        searchable
+                                    >
+                                    </oui-select>
+                                </oui-field>
+
+                                <oui-field
+                                    data-ng-if="$ctrl.isServiceInUK"
+                                    label="{{:: 'telephony_service_contact_outwardcode' | translate }}*"
+                                    error-messages="{ 'required': ('telephony_service_contact_must_fill' | translate) }"
+                                    data-ng-class="$ctrl.isServiceInUK ? 'col-md-4' :  'col-md-12'"
```

Comment:
```suggestion
                                    data-ng-if="$ctrl.isServiceInUK"
                                    label="{{:: 'telephony_service_contact_outwardcode' | translate }}*"
                                    error-messages="{ 'required': ('telephony_service_contact_must_fill' | translate) }"
                                    class="col-md-4"
```

you are data-ng-if so this contition on class isn't required